llvm-external-projects/iree-dialects/test/Dialect/iree_linalg_ext/convert_to_loops.mlir

```diff
@@ -760,3 +760,56 @@ func.func @topk_memref_optional(%input_values: memref<2x10xf32>, %out_values: me
 // CHECK:               %[[D13:.+]] = arith.select %[[D5]], %[[D3]], %[[ARG6]] : f32
 // CHECK:               %[[D14:.+]] = arith.select %[[D10]], %[[D4]], %[[ARG7]] : i32
 // CHECK:               scf.yield %[[D13]], %[[D14]] : f32, i32
+
+// -----
+
+func.func @NC_to_NCNC(%arg0: memref<128x256xf32>, %arg1: memref<4x8x32x32xf32>) -> memref<4x8x32x32xf32> {  
+  iree_linalg_ext.pack ins(%arg0: memref<128x256xf32>) outs(%arg1: memref<4x8x32x32xf32>) inner_tiles [32, 32]
```

Comment:
I'd go for something similar to:
```
iree_linalg_ext.pack %arg0 inner_tiles [32, 32]:  tensor<128x256xf32> -> tensor<4x8x32x32xf32>
iree_linalg_ext.pack %arg0 inner_tiles [32, 32] into %out: (memref<128x256xf32>, memref<4x8x32x32xf32>)
```

or a better spelling .. tensor is not DPS, memref takes the separately allocated buffer